packages/@o3r/apis-manager/src/apis-manager/api-factory.service.ts

```diff
@@ -68,7 +68,7 @@ export class ApiFactoryService {
       ...Object.entries(newItems)
         .reduce<Record<string, Api>>((acc, [apiName, api]) => {
           // eslint-disable-next-line new-cap
-          acc[apiName] = this.isApiClass(api) ? new api(this.getConfigFor(api)) : api;
+          acc[apiName] = this.isApiClass(api) ? new (api)(this.getConfigFor(api)) : api as Api;
```

Comment:
Why do you have to add the cast here ?
`api` should be of type `Api | ApiClassType` but thanks to the type guard of `isApiClass` it could be only `Api`

Reply:
+ the new parentheses should not be mandatory